client/src/components/TextInputComponent/TextInput.css

```diff
@@ -0,0 +1,6 @@
+input {
```

Comment:
Please give the input a specific class name. If not this code will affect all inputs across the entire project.

Reply:
components/SelectDropdown/SelectDropdown.js
Shouldn't be included, i'm sorry.